Framework/DataHandling/src/LoadTOFRawNexus.cpp

```diff
@@ -282,7 +282,6 @@ void LoadTOFRawNexus::loadBank(const std::string &nexusfilename, const std::stri
   file.openGroup("instrument", "NXinstrument");
   file.openGroup(bankName, "NXdetector");
 
-  size_t m_numPixels = 0;
```

Comment:
Can you confirm that you've checked that the removal of these two shadow variables does not impact the behaviour of the code?

Does this need a release note?

Reply:
Sorry, we should have just removed the type `size_it`. But also the code's behaviour won't change as m_numPixels last use in `loadBank` and is not used in the further steps of the algorithm. We will check the unit test again and confirm